ffi/capi/Cargo.toml

```diff
@@ -89,6 +89,9 @@ properties = ["dep:icu_properties", "dep:icu_collections", "dep:unicode-bidi"]
 segmenter = ["dep:icu_segmenter"]
 timezone = ["calendar"]
 
+# Unstable features on individual components
+unstable = ["icu_calendar?/unstable"]
```

Comment:
Should we document the new feature elsewhere, or just here in a cargo.toml comment?

Reply:
No, I think here is sufficient. We don't really document features elsewhere.